### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by joffreynk**
###  Bookstore: styling
styled all components based on design given

### CODE REVIEW

File: src/styles/index.css [hunk 1]
```diff
@@ -7,7 +7,7 @@
 }
 
 body {
-  background-color: rgb(212, 212, 212);
+  background-color: rgb(241, 241, 241);
```
Comment (youmari) [inline src/styles/index.css]: - [ ] please make sure to use the right background color in order to match the Design `#fafafa`

File: src/component/Header.js [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
-    </nav>
+      <nav>
+        <Link to="/">BOOKS</Link>
+        {' '}
+        |
+        {' '}
+        <Link to="categories">CATEGORIES</Link>
+      </nav>
+    </div>
+    <FaUserAlt className="styleIcon" />
```
Comment (youmari) [inline src/component/Header.js]: - [ ] please try to make the account icon a little bit small in order to match the design 
- Yours 
image
- Design 
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @JoffreyNK , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great title and short summary 
✔️ No linter error
✔️ Git flow 

## Required Changes ♻️

- [ ] please try to use the right border color for inputs that add boos in order to match the design 
- Yours 
image
- Design 
image



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @JoffreyNK, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**